### DISCUSSION TITLE
How to make iOS splashscreen fit?

### DISCUSSION THREAD

**Discussion by Ghostbird** [dotnet/maui#13322] [Q&A]
I've got a MAUI splash screen in SVG format that measures 512px × 512px.

```svg
<svg
   width="512"
   height="512"
   viewBox="0 0 512 512"
   version="1.1">
…
</svg>
```

On Android 13 this works fine. But on the iPhone we use for testing, it shows up larger than the screen width (720px).

I've added the `Resize=True` option, but it makes no discernible difference, on either platform.

**Comment by davidortinau** [comment]
Without seeing what your SVG looks like and the current result, it's hard to prescribe a solution. I suspect you will want to set a BaseSize and maybe even reduce the size of your image. Note the resizing for density rules described here in the docs:

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/images/splashscreen?view=net-maui-7.0&tabs=ios

@mattleibow in this case, resizetizer doesn't shrink the image to fit, right? What would be the recommended for the max starting size on a Splash?

Also if you open a launch storyboard in Xcode you'll see that the width of the largest iPhone is 430, which is smaller than your image.

<img width="604" alt="Screenshot 2023-02-16 at 9 07 51 AM" src="https://user-images.githubusercontent.com/41873/219404974-ceac25a7-7a24-4571-a0e2-64a38ce97d02.png">

I believe the MAUI launch storyboard template takes your image and places it in the center of the view, so more suitable for a single logo there. If you want to do something more elaborate and have more control over it, perhaps consider making your own LaunchScreen.storyboard in your solution's Platforms/iOS and adding a reference to it in your info.plist.

**Reply by Ghostbird** [reply]
Thanks for your reply.

I had read the docs, but didn't see anything I did wrong. I tried all possible combinations of `Resize=True` and `BaseSize=256,256`.

I guess I'll try to scale down the entire SVG.

I'm surprised that the maximum width cited by Apple is 430, which I find extremely hard to believe. Wikipedia cited that our smallish older iPhone has a width of 720px.

I suspect that the units Apple uses are not pixels, and that we have a disconnect here, where the SVG values are pixels for all devices except for iOS. Then how can it even work? Such a unit conversion issue reminds me of the [Mars Orbiter loss](www.cnn.com/TECH/space/9909/30/mars.metric.02/index.html).

OFF-TOPIC: I find it funny that in various articles about the orbiter loss, it is described as a _metric error_ or an _imperial error_ depending on which system the writer views as inferior.

**Reply by Ghostbird** [reply]
Tried it today with an SVG that was scaled down to 256px×256px but it shows up exactly the same on the iPhone.
